sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -36,12 +36,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
+      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
```

Comment:
This change and the version of core above are not the correct changes. Dependencies built outside of an area should only ever use the dependency version of a library. If the current version is needed then there should be an unreleased tag and that should exist there until that version is released. This would apply to all of the changes below where dependency was changed to current.

Reply:
`{x-version-update;unreleased_com:azure-core-http-netty;current}` the right tag to use for storage then?

Reply:
Got it to work, I was missing the unreleased tag in version_client.txt